app/src/main/res/values/dimens.xml

```diff
@@ -485,4 +485,10 @@
 
   <dimen name="home_fragment_padding_top">36dp</dimen>
   <dimen name="home_fragment_padding_bottom">148dp</dimen>
+
```

Comment:
Introduce width here
This can be removed as its same across all files. You can keep this only in `values/dimens.xml` file.

Reply:
Removed from other dimens files and kept it in only `values/dimens.xml`